content/en/blog/_posts/2020-08-03-kubernetes-1-18-release-interview.md

```diff
@@ -0,0 +1,215 @@
+
+And I think this discussion around the sidecar enhancement proposal has been very useful, and it points us to the need for more standardized mentoring programs. This is something that multiple SIGs are working on. For example, SIG Contribex, SIG Cluster Lifecycle, SIG Release. The idea is to standardize some sort of mentoring experience so that we can better prepare new contributors to become reviewers and ultimately approvers.
+
+Because ultimately at the end of the day, if we have more people who are knowledgeable about Kubernetes, or even some specific area of Kubernetes, we can better distribute the load, and we can better collaborate on whatever new things come up. I think the sidecar enhancement has shown us mentoring is something worthwhile, and we need a lot more of it. Because as much work as we do, more things are going to continue popping in throughout the project. And the more people we have who are comfortable working in these really complicated areas of Kubernetes, the better off that we are going to be.
+
+**ADAM GLICK: Was there any talk of delaying 1.18 due to the current worldwide health situation?**
+
+JORGE ALARCÓN: We thought about it, and the plan was to just wait and see how people felt. Tried make sure that people were comfortable continuing to work and all the people were landing in new enhancements, or fixing tests, or members of the release team who were making sure that things were happening. We wanted to see that people were comfortable, that they could continue doing their job. And for a moment, I actually thought about delaying just outright— we're going to give it more time, and hopefully at some point, things are going to work out.
+
+But people just continue doing their amazing work. There was no delay. There was no hitch throughout the process. So at some point, I just figured we stay with the current timeline and see how we went. And at this point, things are more or less set.
+
+**ADAM GLICK: Amazing power of a distributed team.**
+
+JORGE ALARCÓN: Yeah, definitely.
+
+[LAUGHING]
+
+**ADAM GLICK: [Taylor Dolezal was announced as the 1.19 release lead](https://twitter.com/alejandrox135/status/1239629281766096898). Do you know how that choice was made, and by whom?**
+
+JORGE ALARCÓN: I actually got to choose the lead. The practice is the current lead for the release team is going to look at people and see, first off, who's interested and out of the people interested, who can do the job, who's comfortable enough with the release team, with the Kubernetes community at large who can actually commit the amount of hours throughout the next, hopefully, three months.
+
+And for one, I think Taylor has been part of my team. So there is the release team. Then the release team has multiple subgroups. One of those subgroups is actually just for me and my shadows. So for this release, it was Mr. Bobby Tables and Taylor. And Taylor volunteered to take over 1.19, and I'm sure that he will do an amazing job.
```

Comment:
```suggestion
And for one, I think Taylor has been part of my team. So there is the release team. Then the release team has multiple subgroups. One of those subgroups is actually just for me and my shadows. So for this release, it was MrBobbyTables and Taylor. And Taylor volunteered to take over 1.19, and I'm sure that he will do an amazing job.
```
Fixing my handle 😆 
One other potential idea since it mixes both handles and names, link out to the GitHub profiles?
```suggestion
And for one, I think Taylor has been part of my team. So there is the release team. Then the release team has multiple subgroups. One of those subgroups is actually just for me and my shadows. So for this release, it was [MrBobbyTables](https://github.com/mrbobbytables) and [Taylor](https://github.com/onlydole). And Taylor volunteered to take over 1.19, and I'm sure that he will do an amazing job.
```

Reply:
We don't generally link names that are mentioned, else it would be blue links everywhere; I've fixed the casing to match the GItHub ID.